x/blobstream/types/validator.go

```diff
@@ -135,7 +137,11 @@ func (ibv InternalBridgeValidators) PowerDiff(c InternalBridgeValidators) float6
 		delta += math.Abs(float64(v))
 	}
 
-	return math.Abs(delta / float64(math.MaxUint32))
+	decDelta := sdk.NewDec(int64(delta))
```

Comment:
could we leave delta a int64 the entire time?

Reply:
@evan-forbes Is the concern related to the `float64` conversion in the earlier lines? If so, then the short answer is yes, we can keep `delta` as `int64` the entire time.

The reason `delta` has been defined as `float64` is because `math.Abs()` accepts and outputs `float64` values. 

If the intention is to keep `delta` as an `int64`, there are two alternative approaches to achieve that:

1. Maintain `delta` as an `int64`, but continue using `math.Abs()` by converting individual powers to `float64`:
```go
var delta int64
for _, v := range powers {
    // NOTE: We are interested in the absolute value of the changes
    delta += int64(math.Abs(float64(v)))
}
```

2. Completely avoid the `float64` conversion of the powers by not using `math.Abs()`. However, please be aware that this approach may have undefined behavior in handling corner cases like when `v` is equal to ±Inf or NaN (which `math.Abs()` can handle: [Link](https://pkg.go.dev/math#Abs)):
```go
var delta int64
for _, v := range powers {
    // NOTE: We are interested in the absolute value of the changes
    if v < 0 {
        v = -v
    }
    delta += int64(v)
}
```
I lean towards option 2, only if the corner cases I mentioned are not conceivable. 
cc: @SweeXordious

Reply:
yeah option 2 is what I think would be preferable